services/accelerometer.md

```diff
@@ -7,6 +7,18 @@
 
 A 3-axis accelerometer.
 
+## Orientation
+
+An accelerometer module should translate acceleration values as follows:
+
+| Orientation           	| X value 	| Y value  	| Z value 	|
+|-----------------------	|---------	|----------	|---------	|
+| Module lying flat     	| 0       	| 0        	| -1000   	|
+| Module on "left" edge   	| -1000   	| 0        	| 0       	|
+| Module on "bottom" edge 	| 0       	| 1000     	| 0       	|
+
+We recommend an orientation marking on the PCB so that users can mount modules without having to experiment with the device.
```

Comment:
should be -1/1 not -1000/1000 (unit is g)

I would also say "module standing upright on "left" edge" to be more precise?

Also some wording like: "left/bottom can be determined by assuming text on silk runs left-to-right"

Reply:
Thanks! will edit